README.md

```diff
@@ -74,6 +74,8 @@ manually if your distribution does not provide a package.
 
 First, make sure you have the required __dependencies__:
 - build:
+	- autopoint (for bootstrapping)
```

Comment:
Which distro do we follow then for listing the dependencies? I feel like this is becoming a wild mix. Maybe make separate sections for different distros?